src/gwt/src/org/rstudio/core/client/theme/res/themeStyles.css

```diff
@@ -230,7 +232,8 @@ select {
 
 .macintosh select {
    font-size: 11px;
-   text-indent: 0.1em;
```

Comment:
Using text-indent rather than padding was causing the text to be cut off in some dialogs -- e.g. note how the `s` in the `Quotes & Brackets` text is cut off a bit.

<img width="347" alt="Screenshot 2024-03-19 at 4 27 21 PM" src="https://github.com/rstudio/rstudio/assets/1976582/012a9723-c1c4-4a56-b174-7909d52db75d">